userguide/services.rst

```diff
@@ -203,55 +199,6 @@ This command can take some time, depending upon the size of the pool or
 dataset being shared. The CNID database is wiped and rebuilt from the
 CNIDs stored in the AppleDouble files.
 
-#ifdef truenas
-.. index:: Asigra
-.. _Asigra DS-System:
-
-Asigra DS-System
-----------------
-
-Asigra Backup allows administrators to back up data from network-connected
```

Comment:
This paragraph should be integrated with the Asigra plugin description.

Reply:
Done: https://github.com/freenas/freenas-docs/pull/1401